src/tequila/apps/krylov/krylov.py

```diff
@@ -0,0 +1,47 @@
+import copy
+import scipy
+import tequila as tq
+from tequila import braket
+from tequila.hamiltonian.qubit_hamiltonian import QubitHamiltonian
+
+
+def krylov_method(krylov_circs:list, H:QubitHamiltonian, variables:dict=None)->tuple:
```

Comment:
can you add some optional arguments (i,e. *args, **kwargs) for flexibility (see comments below)